neutrino-custom/base.js

```diff
@@ -0,0 +1,189 @@
+'use strict';
+const path = require('path');
+const webpack = require('webpack');
+const reactPreset = require('neutrino-preset-react');
+const HtmlPlugin = require('html-webpack-plugin');
+const Md5HashPlugin = require('webpack-md5-hash');
+
+const CWD = process.cwd();
+const SRC = path.join(CWD, 'src'); // neutrino's default source directory
+const UI = path.join(CWD, 'ui');
+const DIST = path.join(CWD, 'dist');
+const INDEX_TEMPLATE = path.join(UI, 'index.html');
+const ADMIN_TEMPLATE = path.join(UI, 'admin.html');
+const PERF_TEMPLATE = path.join(UI, 'perf.html');
+const LOGVIEWER_TEMPLATE = path.join(UI, 'logviewer.html');
+const FAILUREVIEWER_TEMPLATE = path.join(UI, 'failureviewer.html');
+const USERGUIDE_TEMPLATE = path.join(UI, 'userguide.html');
+
+const HTML_MINIFY_OPTIONS = {
+    useShortDoctype: true,
+    keepClosingSlash: true,
+    collapseWhitespace: true,
+    preserveLineBreaks: true
+};
+
+module.exports = neutrino => {
+    reactPreset(neutrino);
+
+    // Change the ouput path from build/ to dist/:
+    neutrino.config.output.path(DIST);
+
+    if (process.env.NODE_ENV !== 'test') {
+        // Include files from node_modules in the separate, more-cacheable vendor chunk:
```

Comment:
Rather than having to keep the list of vendor deps up to date, would something based on filepath like this work?
https://webpack.js.org/guides/code-splitting-libraries/#implicit-common-vendor-chunk